librz/core/cmd/cmd_analysis.c

```diff
@@ -2501,60 +2388,67 @@ static void cmd_analysis_esil(RzCore *core, const char *input) {
 	}
 }
 
-static void cmd_analysis_jumps(RzCore *core, const char *input) {
-	rz_core_cmdf(core, "af @@= `axl~ref.code.jmp[1]`");
-}
-
-// TODO: cleanup to reuse code
-static void cmd_analysis_aftertraps(RzCore *core, const char *input) {
-	int bufi, minop = 1; // 4
-	ut8 *buf;
+/**
+ * \brief      Prints commands to create functions after traps calls
+ *
+ * \param[in]  core    The RzCore to use
+ * \param[in]  n_bytes The number of bytes after the trap calls (set to 0 for auto-mode)
+ *
+ * \return    False on failure, otherwise true
+ */
+RZ_API bool rz_core_analysis_after_traps_print(RZ_NONNULL RzCore *core, ut64 n_bytes) {
```

Comment:
If you add `RZ_NONNULL` please also add an assert.

Reply:
Moreover, please move this function to `librz/core/canalysis.c` since it's an API. If you think it's necessary only for commands - maybe it's better to make it RZ_IPI then?